### ISSUE TITLE
Cannot pass CI with pre-existing API spec that lacks tenancy params 

### ISSUE THREAD

**Issue by cmars** [snyk/sweater-comb#136]
When introducing Optic CI into an existing project, I found this rule failing with no changes:

```
  FAIL   This Specification
  x this specification:  must have an org or group tenant
    expected support for org or group tenant: expected [] to have a length above 0 but got 0
    Read more in our API guide (​https://github.com/snyk/sweater-comb/blob/main/docs/standards.md#organization-and-group-tenants-for-resources​)
    at (/Users/cmars/Projects/registry/src/web/routes/api/v3/resources/_examples/hello_world/2021-06-13/spec.yaml:1:0)
```

The paths are: `/examples/hello_world`, `/examples/hello_world/{id}`. They aren't in compliance, but they're also not changed in the CI diff.

We want to improve these, but if they've already landed, they shouldn't block introduction of Optic CI into the project. Tenancy rules may also need to evolve over time...

**Comment by acunniffe** [comment]
Hola! Back from break, hope yours was good. 

Since we're talking about checking for things that need to be present, we have to use `requirement`. We could do more tuning to prevent this from being triggered in this instance, but I'm feeling a strong pull towards the `effectiveOn` idea you shared in December that allows old versions to be exempt from some rules. 

> Tenancy rules may also need to evolve over time...

your callout here reinforces this direction from my POV. 

Should I move ahead and have the team open some PRs for this?

**Comment by cmars** [comment]
Welcome back! It was a good break, hope yours was as nice and relaxing!

Yes please re: `effectiveOn`; that should solve this particular problem nicely, and lets us confidently evolve our rules moving forward.

**Comment by cmars** [comment]
Fixed by #140
